incubator/java-openliberty/image/project/Dockerfile

```diff
@@ -12,23 +12,29 @@ RUN cd /project && mvn -B install dependency:go-offline -DskipTests
 COPY ./util /project/util
 RUN  /project/util/check_version build
 
-# Copy the user's application pom and resolve all dependencies, also trigger Open Liberty install
+# Download Openliberty here to prevent a re-download
+RUN cd /project && mvn -B liberty:install-server liberty:create
```

Comment:
I think we might run into issues installing/creating the server here, using the parent pom. OL will be installed in /project/target which the app (user-app/pom.xml) doesn't know about. I think it would be better to do this install later, after we copy the user-app/pom.xml to OL is installed/created in the child's target dir (user-app/target).